tests/everest/test_detached.py

```diff
@@ -48,7 +48,12 @@
 @pytest.mark.fails_on_macos_github_workflow
 @pytest.mark.xdist_group(name="starts_everest")
 async def test_https_requests(copy_math_func_test_data_to_tmp):
-    everest_config = EverestConfig.load_file("config_minimal_slow.yml")
+    everest_config = EverestConfig.load_file("config_minimal.yml")
+    everest_config.install_jobs.append(
+        InstallJobConfig(name="sleep", source="jobs/SLEEP")
+    )
+    everest_config.forward_model.append("sleep --sleep 10")
+
```

Comment:
Looks good, but could probably be simplified even more, as we dont really need the full math func case to test this, for example:

```suggestion
async def test_https_requests(tmp_path, monkeypatch, min_config):
    monkeypatch.chdir(tmp_path)
    with open("sleep_job", "w", encoding="utf-8") as fout:
        fout.write("EXECUTABLE sleep")
    with open("config.yml", "w") as fout:
        yaml.dump(min_config | {
            "install_jobs": [{"name": "sleep", "source": "sleep_job"}],
            "forward_model": ["sleep --sleep 10"],
        }, fout)

    everest_config = EverestConfig.load_file("config.yml")
```

 where: min_config is a fixture in: tests/everest/test_everlint.py, which can be moved to a shared `conftest.py`
that way we could probably delete the `SLEEP` job and `sleep.py`, as it does not really make sense to have that checked in as a part of an example.

Can also keep as it, but suggest removing `sleep.py` and `jobs/SLEEP` regardless, as they are not relevant for an acual case, it is just for testing.

The other parts of the config is not really relevant for this test anyway.

Reply:
Great suggestion, thanks :)!